src/tx/applyTx/checkExit.js

```diff
@@ -26,7 +26,8 @@ module.exports = (state, tx, bridgeState) => {
     !exit ||
     !addrCmp(exit.exitor, unspent.address) ||
     !equal(BigInt(exit.amount), BigInt(unspent.value)) ||
-    Number(exit.color) !== unspent.color
+    Number(exit.color) !== unspent.color ||
+    exit.data !== unspent.data
```

Comment:
what about NFT exits that don't have data. will the comparison of null !== null work?